generate-stackbrew-library.sh

```diff
@@ -2,8 +2,9 @@
 set -Eeuo pipefail
 
 declare -A aliases=(
-	[7.4]='7 latest'
-	[8.0-rc]='rc'
+	[7.4]='7'
+	[8.0]='8 latest'
+	[8.1-rc]='rc'
```

Comment:
PHP 8.1 has no release candicated. Would it be possible to release devel (php-src master) nighty?